File: db4oj.tests/src/com/db4o/db4ounit/common/events/AllTests.java
/* Copyright (C) 2007   db4objects Inc.   http://www.db4o.com */
package com.db4o.db4ounit.common.events;

import db4ounit.extensions.*;

public class AllTests extends Db4oTestSuite {

	protected Class[] testCases() {
		return new Class[] {
			ActivationEventsTestCase.class,
			ClassRegistrationEventsTestCase.class,
			DeletionEventsTestCase.class,
			InstantiationEventsTestCase.class,
		};
	}

}

File: db4o.net/Db4objects.Db4o.Tests/Db4objects.Db4o.Tests/Common/Acid/AllTests.cs
using System;
using Db4oUnit.Extensions;
using Db4objects.Db4o.Tests.Common.Acid;

namespace Db4objects.Db4o.Tests.Common.Acid
{
	public class AllTests : Db4oTestSuite
	{
		protected override Type[] TestCases()
		{
			return new Type[] { typeof(CrashSimulatingTestCase) };
		}
	}
}
